tests/molecular/molecules/molecule/fixtures/host_guest/complex.py

```diff
@@ -51,6 +53,25 @@
                 '+]4[CH+][C+2]C4([C+2]2)C2=C6[C+]=[C+]2)[C+]13.[H]C#N'
             ),
         ),
+        CaseData(
```

Comment:
did you commit the expected position matrix for these test cases? (Otherwise its always gonna pass in CI)

Reply:
Did not even think of this because I no longer run local tests. That is a problem. Solving now.